spec/rest_in_peace/definition_proxy/resource_method_definitions_spec.rb

```diff
@@ -127,14 +127,14 @@
   context '#put' do
     it_behaves_like 'an instance method' do
       let(:http_verb) { :put }
-      let(:method_name) { :update }
+      let(:method_name) { :update_a }
```

Comment:
Ah, maybe you can give it a meaningful name? Someone else looking at the code might be confused.